packages/@aws-cdk/aws-cloudfront/package.json

```diff
@@ -50,6 +50,11 @@
   "cdk-build": {
     "cloudformation": "AWS::CloudFront"
   },
+  "awslint": {
+    "exclude": [
+      "props-physical-name:@aws-cdk/aws-cloudfront.CloudFrontOriginAccessIdentityProps"
```

Comment:
I used just `name` instead of `cloudFrontOriginAccessIdentityName`

For example:
```
const oai = cloudfront.CloudFrontOriginAccessIdentity.fromName(
  stack,
  'OAIImported',
  'AEFASDF234'
);
```

instead of

```
const oai = cloudfront.CloudFrontOriginAccessIdentity.fromCloudFrontOriginAccessIdentityName(
  stack,
  'OAIImported',
  'AEFASDF234'
);
```

Reply:
Our convention is to use the full name (See other L2s)

Reply:
Please remove all excludes related to this resource.